### PR TITLE
Add code html and style

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://juajum.github.io/layout_snake/)
- [TEST REPORT LINK](https://juajum.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (jstmpelowycz) [inline src/index.html:6]: Format code properly

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+
```
Comment (jstmpelowycz) [inline src/index.html:12]: I don't think you need this extra whitespace

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+  height: 300px;
+  width: 300px;
```
Comment (jstmpelowycz) [inline src/style.css:18]:
```suggestion
  min-width: 300px;
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  .block__5 {
+    order: 2;
+  }
+
+  .block__6 {
+    order: 3;
+  }
+}
+
+@media (min-width: 750px) {
```
Comment (jstmpelowycz) [inline src/style.css:56]: You don't need it

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  .block__5 {
+    order: 2;
+  }
+
+  .block__4 {
+    order: 3;
+  }
+}
+
+@media (min-width: 1050px) {
```
Comment (jstmpelowycz) [inline src/style.css:103]: You don't need it

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+  .container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .block {
+    flex-basis: 350px;
+  }
+}
+
+@media (min-width: 1200px) {
```
Comment (jstmpelowycz) [inline src/style.css:113]: You don't need it

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+  .container {
+    flex-direction: column;
+  }
+}
+
+@media (min-width: 450px) {
+  .block {
+    width: 450px;
+  }
+}
```
Comment (jstmpelowycz) [inline src/style.css:31]: You don't need it

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
There are a lot of redundant media queries, they cause your layout to be like that
Screenshot 2022-05-11 at 21 14 47
Screenshot 2022-05-11 at 21 13 35
Screenshot 2022-05-11 at 21 13 17
Screenshot 2022-05-11 at 21 12 50

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**